### PR TITLE
Add test config

### PR DESCRIPTION
**Pull request by luigibk**
### JIRA link (if applicable) ###

https://dev.azure.com/hmcts/Platform%20Engineering/_workitems/edit/19

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/aat/software-engineering/data-extractor-job/data-extractor-values.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: data-extractor-values
+  namespace: software-engineering
+spec:
+  encryptedData:
```
Comment (timja) [inline k8s/aat/software-engineering/data-extractor-job/data-extractor-values.yaml]: what's in here?

File: k8s/aat/software-engineering/data-extractor-job/data-extractor-job.yaml [hunk 2]
```diff
@@ -13,7 +13,28 @@ spec:
...
+  - name: "data-extractor-values"
   values:
     job:
       image: hmcts.azurecr.io/hmcts/data-extractor-job:prod-f888e665
       suspend: true
+      keyVaults:
+        "rpe-prod":
+          resourceGroup: rpe-prod
+          secrets:
+            - aat-ccdro-user
```
Comment (timja) [inline k8s/aat/software-engineering/data-extractor-job/data-extractor-job.yaml]: this shouldn't be retrieved from rpe-prod, I only suggested it as somewhere for it to put so we can receive it.

A temp MI vault should be created, once the MI subscription is setup for its AKS cluster the secret can be moved there

File: k8s/aat/software-engineering/data-extractor-job/data-extractor-values.yaml (hunk 1 above)
Comment (luigibk) [reply k8s/aat/software-engineering/data-extractor-job/data-extractor-values.yaml]: env that cannot be cleartext

Comment (timja) [reply k8s/aat/software-engineering/data-extractor-job/data-extractor-values.yaml]: which variables specifically?

Comment (timja) [reply k8s/aat/software-engineering/data-extractor-job/data-extractor-values.yaml]: i.e you didn't answer the question... 👅

**Pull request merged**